themes/default/layouts/shortcodes/langfile.html

```diff
@@ -2,7 +2,7 @@
 <pulumi-chooser type="language" options="javascript,typescript,python,go,csharp,fsharp,visualbasic" option-style="none" class="inline">
 <pulumi-choosable type="language" value="javascript"><code>index.js</code></pulumi-choosable>
 <pulumi-choosable type="language" value="typescript"><code>index.ts</code></pulumi-choosable>
-<pulumi-choosable type="language" value="python">`__main__.py`</pulumi-choosable>
+<pulumi-choosable type="language" value="python"><code>__main__.py</code></pulumi-choosable>
```

Comment:
Odd; I had switched it to use the markdown because the dunders weren't appearing properly (they were causing things to get underlined, instead). I'm not sure why they're suddenly appearing properly 🤔  Has there been another change somewhere?

Reply:
Yes, I went ahead and fixed this to align with the others because it was showing up as ticks in other areas of the site:

![image](https://user-images.githubusercontent.com/274700/153299082-6f295373-1397-403b-8739-00b80bb0a70a.png)

The gist is that Hugo will render the content of shortcodes called with `{{% %}}`, but not for `{{< >}}`, which is why we see it rendered properly in Learn but not in the Getting Started guide. There are a few places where we call `langfile` with `%`s, so those'll need to be fixed up as well (one of which is my last blog post 🤦 -- I'll do that in this PR also), but as long as we use `{{< langfile >}}` going forward, we should be good.

Reply:
Ahhhhhh, so that makes sense. I guess the other pages I checked when I first checked the preview on that PR must have used the other version that I was attempting to use. So Hugo renders ones with % % as markdown inside, but the ones with < > as standard code with a markdown override?

Reply:
I _think_ so, if I understand you correctly. The gist is that with percents, you're saying, "run this content through the Markdown renderer". Otherwise it just treats content as regular HTML.

Reply:
Ha, sorry. Words, ya know? That's exactly what I meant, so it makes sense :)